### ISSUE TITLE
Is this where I can raise my questions？

### ISSUE THREAD

**Issue by DarkBK201** [Zhoub-BNU/math_phys#8]

**Comment by bnuzhoub** [comment]
Yes.  But you should first prove that you are one of the students in the course, as shown in the file [involving.md](https://github.com/Zhoub-BNU/math_phys/blob/master/involving.md).
**It has been updated.**
